tests/test_stations.py

```diff
@@ -3,16 +3,37 @@
 import unittest
 
 from openkongqi.utils import get_uuid
+from openkongqi.stations import _STATIONS_MAP, get_station_map
```

Comment:
In this test, you use production data that might change in the future and break your test. You have to create test data as I showed you yesterday in `tests/data/stations`, then look at `tests/test_fetch.py` on how to feed a crafted configuration object to provide only `settings['STATIONS_MAP_DIR']` that points to your test folder data.

I also notice that you import `_STATIONS_MAP` but not use it later.